src/Report/main.tex

```diff
@@ -447,11 +448,22 @@
     
     \clearpage
     \section{Zusammenfassung}
-        \subsection{Verbesserungen am Fragebogen}
-            % TODO
-            \begin{itemize}
-                \item In Umfrage sollte man nicht "Ja/Nein" sondern "Ja/Nein/Keine Ahnung" machen. Evtl. auch die maximal mögliche Signifikanz dessen erörtern.
-            \end{itemize}
+        \subsection{Selbstreflexion}
+            Im folgenden wird kritisch auf das Vorgehen in dieser Arbeit eingegangen und mögliche Methoden zur Optimierung der Ergebnisse dargelegt.
+            \paragraph{Wann ist eine Software Open Source?}
+                In der Umfrage wurde lediglich erwähnt, dass eine Software im Sinne der Umfrage auch als Open Source gilt, wenn sie \gls{oss} beinhaltet. Wie bereits Eingangs beschrieben wurden genutzte Programmiersprachen oder Datenbanken nicht als signifikante Open Source Komponente betrachtet. Dies wurde aus dem Fragebogen nicht ersichtlich und man hätte für bessere Umfrageergebnisse genauer spezifizieren müssen, dass eine Anwendung als Open Source gilt, wenn sie selbst quelloffen ist oder ihre Funktionalität durch die Entfernung von Open Source Komponenten signifikant eingeschränkt wäre.
+                
+            \paragraph{Enthaltung} % TODO note that we only used the ones where the users used the program
+                Da der Fragebogen bei der Anwendungsmatrix lediglich die Optionen \emph{Ja} und \emph{Nein} beinhaltete besteht die Möglichkeit, dass die Befragten die Felder auf ihrem Standardwert ({\scriptsize Nein}) belassen haben, wenn sie sich unsicher waren. Dies hätte man umgehen können, indem man eine dritte Option für das Feld einfügt, welche Enthaltung repräsentiert. Um der potentiellen Verfälschung der Ergebnisse entgegenzuwirken wurden in dieser Arbeit lediglich die Datenpunkte aus der Matrix in Betracht gezogen, wo angegeben wurde, dass die Anwendung auch verwendet wird.
```

Comment:
Ich finde Formulierungen mit wo irgendwie merkwürdig. Ich würde das so schreiben: `die Datenpunkte aus der Matrix der Nutzer in Betracht gezogen, die auch abgaben, die Anwendung zu verwenden`.